### PR TITLE
My contact

### PR DESCRIPTION
**Pull request by tonnymuchui**
worked on the footer section html&css. where I added contact information and fixed linters error spaces

### CODE REVIEW

**Comment by tonnymuchui** [comment]
worked on the footer section html&css. where I added contact information

File: index.html [hunk 1]
```diff
@@ -241,6 +241,66 @@ <h3 class="header-text">About Myself</h3>
...
+                  required="required"
+                ></textarea>
+              </li>
+              <li>
+                <button class="getin getin1" type="submit">Get It Touch</button>
+              </li>
+            </ul>
+          </form>
+        </div>
+      </footer>
```
Comment (tresorsawasawa) [inline index.html:303]: - [ ] Kindly make sure you also add the missing **Horizontal** line at the bottom of the page to stick to the Figma design.

| Figma Design | Your Design |
| -------------------- | ---------------------- |
| Screenshot from 2022-08-24 18-19-39 | Screenshot from 2022-08-24 18-19-07 |

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @tonnymuchui 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tonnymuchui** [comment]
Working on them right now. Thank you.

**Comment by tonnymuchui** [comment]
added a horizontal line to the footer & fixed submit button

**Review by awais-amjed — APPROVED** [review]
# STATUS: APPROVED 🥳

Hi @tonnymuchui , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations on implementing the requested changes! 🎉👏🥇✨

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] No linter issues ✔️

gif

## Optional suggestions

- [ ] It would be nice if you can fix the spacing around your website and between different elements to match the Figma design 🤗

Your design:
image

Figma design:
image

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**